R/importMetaphlan.R

```diff
@@ -124,10 +124,8 @@ importMetaPhlAn <- function(
     tables <- .parse_metaphlan(data, ...)
 
     # Create multiple SE objects at different rank from the data
-    available_ranks <- names(tables)
     se_objects <- lapply(tables, function(x){
-        .create_se_from_metaphlan(
-            x, rowdata_col, returned.ranks = available_ranks, ...)
+        .create_se_from_metaphlan(x, rowdata_col, ...)
         })
```

Comment:
`returned.ranks` controls which taxonomy ranks are included in rowData(tse). There is no really user-case for controlling it since we want all the data.

Reply:
Also, I believe controlling it does not affect the results since the ranks should already match because they are got from the metaphlan file...